packages/frontend/src/ee/features/embed/EmbedDashboard/components/EmbedDashboard.tsx

```diff
@@ -174,66 +324,74 @@ const EmbedDashboard: FC<{
-                                tile={tile}
-                                isEditMode={false}
-                                onDelete={() => {}}
-                                onEdit={() => {}}
-                            />
-                        ) : (
-                            assertUnreachable(
-                                tile,
-                                `Dashboard tile type is not recognised`,
-                            )
-                        )}
-                    </div>
-                ))}
-            </ResponsiveGridLayout>
+
+            {tabsEnabled ? (
+                <Tabs
+                    value={activeTab?.uuid}
+                    onTabChange={(e) => {
+                        const tab = sortedTabs.find((t) => t.uuid === e);
+                        if (tab) {
+                            setActiveTab(tab);
+                        }
+                    }}
+                    mt="md"
+                    styles={{
+                        tabsList: {
+                            flexWrap: 'nowrap',
+                            height: MAGIC_SCROLL_AREA_HEIGHT - 1,
+                        },
+                    }}
+                    variant="outline"
+                >
+                    <Tabs.List bg="gray.0" px="lg">
+                        {sortedTabs.map((tab) => (
+                            <Tabs.Tab
+                                key={tab.uuid}
+                                value={tab.uuid}
+                                bg={
+                                    activeTab?.uuid === tab.uuid
+                                        ? 'white'
+                                        : 'gray.0'
+                                }
+                            >
+                                <Title
+                                    fw={500}
+                                    order={6}
+                                    color="gray.7"
+                                    truncate
+                                    maw={`calc(${
+                                        100 / (sortedTabs?.length || 1)
+                                    }vw)`}
+                                >
+                                    {tab.name}
+                                </Title>
+                            </Tabs.Tab>
+                        ))}
+                    </Tabs.List>
+                    <EmbedDashboardGrid
+                        filteredTiles={filteredTiles}
+                        layouts={layouts}
+                        dashboard={dashboard}
+                        projectUuid={projectUuid}
+                        embedToken={embedToken}
+                        hasRequiredDashboardFiltersToSet={
+                            hasRequiredDashboardFiltersToSet
+                        }
+                        isTabEmpty={isTabEmpty}
+                    />
+                </Tabs>
+            ) : (
+                <EmbedDashboardGrid
```

Comment:
no tabs, we simply show the existing dashboard grid with all tiles (existing behaviour)